.github/workflows/rust.yml

```diff
@@ -277,7 +277,9 @@ jobs:
           rustup default stable
           rustup component add rustfmt
       - name: Run
-        run: ci/scripts/rust_fmt.sh
+        run: |
+          echo '' > datafusion/proto/src/generated/datafusion.rs
```

Comment:
This is needed so it works on a clean checkout, right?

Reply:
The CI jobs are parallelized, so `Lint` doesn't get a copy of the files generated by `Build`, but lint did complain I did a `mod datafusion` for a file that didn't exist. Since this is a CI-only issue, this seemed like an acceptable fix for CI ... ?